kratos/processes/levelset_convection_process.h

```diff
@@ -165,7 +166,7 @@ class LevelSetConvectionProcess
     /// Destructor.
     ~LevelSetConvectionProcess() override
     {
-        mrBaseModelPart.GetModel().DeleteModelPart("DistanceConvectionPart");
+        mrModel.DeleteModelPart("DistanceConvectionPart");
```

Comment:
consider using a name that is depending on the original name, otherwise it is not possible to have multiple instances of this process

Reply:
Nop, I used only one redistance, so name should not matter. I changed it to use original model part name, is that ok?